tck/pom.xml

```diff
@@ -29,6 +29,7 @@
     <description>Eclipse MicroProfile JWT Feature - TCK</description>
 
     <properties>
+        <version.microprofile.config>1.4</version.microprofile.config>
```

Comment:
I think you will need to depend on Config 2.0 since JWT 1.2 and Config 2.0 are under MP 4.0.

Reply:
Yes, this was just until we get the proper MP Config 2.0 release.

Reply:
@Emily-Jiang Yeah, we are waiting for MP Config 2.0, but this is just a minor update for us to check 1.4 (as it should've been set earlier as spotted bu Roberto) is fine as expected, thanks